### PR TITLE
feat(boinc): kaktoos-cuda 1.01 (y=62 height=7 singlechunk)

### PR DESCRIPTION
**Pull request by BoySanic**
Added new binaries for kaktoos (lin and win)

Added new workunit generation for loop for kaktoos

Amended apps and projects xmls to allow for new project.

Added new template file the file size needs of the new tasks, and to free up the stderr for actual errors and compute speeds, etc, viewable on the site.

Changed server_status.php to use proper terminology

### CODE REVIEW

File: images/makeproject/makeproject-step2.sh [hunk 1]
```diff
@@ -67,6 +67,23 @@ yes | bin/update_versions
...
 #bin/create_work --appname kaktwoos --wu_template templates/main_in --result_template templates/main_out -wu_name main_nodelete
 
+#
+# 2^12 boinc work units, 2^36 seeds per kaktoos work unit = 2^48 seeds total, 600kb per result, 128,946,176 final seed count est.
+#for i in {0..4095}; do
+<<<<<<< HEAD
+# wu_name="kaktoos_1.01_y62_7_$((i * 68719476736))"
+=======
+# wu_name="kaktoos_y62_7_$((i * 68719476736))"
+>>>>>>> 934dc20... feat(kaktoos): 1.01
```
Comment (Earthcomputer) [inline images/makeproject/makeproject-step2.sh:77]: Please fix this merge conflict

**Comment by Earthcomputer** [comment]
Just so you know, it's bad practice to force-push in a PR, as it makes it hard to review the changes. Better practice is to have whoever merges it squash the commits.

**Comment by BoySanic** [comment]
> Just so you know, it's bad practice to force-push in a PR, as it makes it hard to review the changes. Better practice is to have whoever merges it squash the commits.

Got it. Yeah I guess I was trying to keep the number of commits to a minimum, as far as I understood it was sort of on me to squash them. If it's okay for me to create new commits in a PR rather than preserving the current number, I can do that. The most recent pushes were a rebase as I included too many commits in my original rebase.

**Comment by Hyenadae** [comment]
We've almost got this, but we need to merge:

https://github.com/BoySanic/minecrafthome/commit/b7faa4abe4d2248b681146e4fe1cf7aa0da1a8d3 

these changes and ensure all python scripts have the right header / env define + executable set on git

**Comment by bscout9956** [comment]
image
BoySanic meant #43

image

**Pull request merged**